scripts/hdf_gt_to_tif.py

```diff
@@ -25,7 +25,9 @@ def get_num_frames(options):
 
 def save_frame_to_tif(timestep, label_image, options):
     if len(options.input_files) == 1:
-        filename = options.output_dir + '/man_seg' + format(timestep, "0{}".format(options.filename_zero_padding)) + '.tif'
+        # filename = options.output_dir + '/man_seg' + format(timestep, "0{}".format(options.filename_zero_padding)) + '.tif' # default, GIT
+        # filename = options.output_dir + '/mask' + format(timestep, "0{}".format(options.filename_zero_padding)) + '.tif' # for letting SEGMEasure run before tracking
+        filename = options.output_dir + '/seg' + format(timestep, "0{}".format(options.filename_zero_padding)) + '.tif' # for converting Phillips prediction pams to segm
```

Comment:
Is there a reason for the change from `/man_seg` to `/seg`? I'd prefer to have the standard CTC naming in here.
Leaving the other options in as comments is fine with me.